docs/UserGuide.md

```diff
@@ -224,6 +232,32 @@ Precise expected output on success:
 Precise expected outputs on failure:
 * Error message shown to the user: "The person already exists in the address book"
 
+### Sorting contacts : `sort`
+
+Sort contact lists by parameter name or appointment date.
+
+Format: `undo PARAMETER`
```

Comment:
Maybe can change the "Format:" into "Precise command format:" as we all have agreed

Reply:
Shouldn't it be "sort Parameter"?

Reply:
done!